jetty-security/src/main/java/org/eclipse/jetty/security/JDBCLoginService.java

```diff
@@ -61,9 +52,9 @@
     protected String _userTableKey;
     protected String _userTablePasswordField;
     protected String _roleTableRoleField;
-    protected Connection _con;
     protected String _userSql;
     protected String _roleSql;
+    protected Connection _con;
 
     /**
      * JDBCKnownUser
```

Comment:
The `JDBCUserPrincipal` class below could be static.

Also the `JDBCLoginService` constructors all say they throw IOException but none of them actually do, so maybe that declaration could be removed as well.